### DISCUSSION TITLE
GitHub Support Has Not Responded Already for 50 Days

### DISCUSSION THREAD

**Discussion by tintinwp** [community/community#129603] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was wondering why GitHub Support hasn't responded even after 50 days ?

I can't get help from the community regarding my issue, which involves my account getting hacked.

How long did it take for me to get a response after creating a ticket on GitHub, especially on my cases ?  Is there any call center on github service ? or something that i can do a follow up ?

My ticket ID: 
#2751235
#2801542

**Comment by LiteBrite82** [comment]
Hi @TinTin6892 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

If you have 2FA set up on your account, you might try the [2FA recovery method](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials)

Thank you!
